matchbox_socket/src/webrtc_socket/socket.rs

```diff
@@ -416,15 +416,34 @@ impl<C: ChannelPlurality> WebRtcSocket<C> {
     /// this method was called.
     ///
     /// See also: [`PeerState`]
+    ///
+    /// # Panics
+    ///
+    /// Will panic if the channel has been closed or is broken.
```

Comment:
Should this say socket is dropped or disconnected from the signaling server?

Reply:
Just thinking "the channel" is an implementation detail and it should rather be explained in api usage terms